app/src/main/java/com/example/albumsapp/parser/PhotoColorParser.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.albumsapp.parser
+
+import android.graphics.Color
+import java.util.Locale
+
+class PhotoColorParser {
+    fun parse(url: String): Int {
+        val color = StringBuilder("#")
+        color.append(url.takeLastWhile { it != '/' }.uppercase(Locale.ROOT))
+        while (color.length < 7) {
+            color.insert(5, "0")
```

Comment:
i'm still not sure this implementation is correct. Let's try to unit test this parser. 
The scenarios I'd recommend you to test are next:
- parse("https://placeholder.com/600/ff") -> should return 255 (or 0xff or blue) 
- parse("https://placeholder.com/600/0") -> should return 0 (0x0 or black) 
- parse("https://placeholder.com/600/ffffff") -> should return 16777215 (0xffffff or white)

Reply:
i think that color in hexadecimal format divided into three parts: red, green, blue. For example if we have color #ff44ff that's mean:
- red = ff 
- green 44
- blue ff
In our case #61A65 equals  #61A605, because 5 == 05 in hexadecimal.

Reply:
For 6 symbols in a color it makes sense and I agree. But if it's shorter than 6 I'm not sure.
Following your logic, will #aaa color be equal to #0a0a0a or #aaa000 ?

Reply:
So this way we can parse any integer into a color

Reply:
https://en.wikipedia.org/wiki/Web_colors
If any one of the three color values is less than 10 hex (16 decimal), it must be represented with a leading zero so that the triplet always has exactly six digits. For example, the decimal triplet 4, 8, 16 would be represented by the hex digits 04, 08, 10, forming the hex triplet 040810.